rollup.config.js

```diff
@@ -56,6 +60,9 @@ export default {
       __VERSION__: execSync("git rev-list HEAD --max-count=1")
         .toString()
         .trim(),
+      "process.env.NODE_ENV": JSON.stringify(
```

Comment:
Let's add a quick note about why we added this

```suggestion
      // workaround the way sveltejs-tippy imports tippy: https://github.com/mdauner/sveltejs-tippy/issues/117
      "process.env.NODE_ENV": JSON.stringify(
```